sdk/translation/azure-ai-translation-text/azure/ai/translation/text/aio/_patch.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+from typing import Union
+from azure.core.credentials import ( TokenCredential, AzureKeyCredential )
```

Comment:
You'll need to break this into two imports

```
from azure.core.credentials import AzureKeyCredential
from azure.core.credentials_async import AsyncTokenCredential
```

Reply:
Done